schema.json

```
@@ -62,90 +62,63 @@
 
 		"configManagementConfig": {
 			"type": "object",
-			"oneOf": [
-				{
-					"required": ["helm"],
-					"additionalProperties": false,
-					"properties": {
-						"helm": {
-							"$ref": "#/definitions/helmConfig"
-						}
-					}
-				},
-				{
-					"required": ["kustomize"],
-					"additionalProperties": false,
-					"properties": {
-						"kustomize": {
-							"$ref": "#/definitions/kustomizeConfig"
-						}
-					}
-				},
-				{
-					"required": ["ytt"],
-					"additionalProperties": false,
-					"properties": {
-						"ytt": {
-							"$ref": "#/definitions/yttConfig"
-						}
+			"oneOf": [{
```

```suggestion
			"anyOf": [{
```

I think this change is needed, but last I checked (yesterday), this wasn't passing, but I think my editor has the old schema cached. Will play with it some more...

---

Thanks for catching issue with schema. Issue is fixed now - added tests that cover required scenarios.